util/src/main/java/tc/oc/pgm/util/menu/InventoryMenu.java

```diff
@@ -0,0 +1,315 @@
+package tc.oc.pgm.util.menu;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static net.kyori.adventure.text.Component.translatable;
+import static tc.oc.pgm.util.menu.InventoryMenuUtils.howManyRows;
+
+import com.google.common.collect.Lists;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+import net.kyori.adventure.text.Component;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.craftbukkit.v1_8_R3.util.WeakCollection;
```

Comment:
We can't depend on this because it's `nms`. Use a `WeakHashMap` with `Boolean` values.